src/hevm/src/EVM/SymExec.hs

```diff
@@ -300,9 +314,13 @@ panicMsg :: Word256 -> ByteString
 panicMsg err = (selector "Panic(uint256)") <> (encodeAbiValue $ AbiUInt 256 err)
 
 verifyContract :: SolverGroup -> ByteString -> Maybe (Text, [AbiType]) -> [String] -> StorageModel -> Maybe Precondition -> Maybe Postcondition -> IO [VerifyResult]
-verifyContract solvers theCode signature' concreteArgs storagemodel maybepre maybepost = do
+verifyContract solvers theCode signature' concreteArgs storagemodel maybepre maybepost =
```

Comment:
Same question as for `checkAssert`. Do we need this variant?

Reply:
Good point, sorry. I didn't want to change all the lines of code in `test.hs`, but it was not a good strategy. We can fix them up differently if they are ugly, e.g. by writing a helper function in `test.hs`.